rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/azure/AzureBakeHandler.groovy

```diff
@@ -61,9 +62,11 @@ public class AzureBakeHandler extends CloudProviderBakeHandler{
     // TODO(duftler): Presently scraping the logs for the image name. Would be better to not be reliant on the log
     // format not changing. Resolve this by storing bake details in redis.
     logsContent.eachLine { String line ->
-      if (line =~ IMAGE_NAME_TOKEN) {
-        imageName = line.split("/").last()
-        ami = "https:" + line.split(":").last()
+      if (line.startsWith(IMAGE_NAME_TOKEN)) {
+        imageName = line.substring(IMAGE_NAME_TOKEN.size())
+      }
+      if (line.startsWith(IMAGE_ID_TOKEN)) {
```

Comment:
Are there two business scenarios mapped to IMAGE_NAME_TOKEN and IMAGE_ID_TOKEN separately?  If so, suggest adding comments here telling what they are and better with sample values of them.